scranShipper/src/Basket/Basket.jsx

```diff
@@ -0,0 +1,68 @@
+import BasketItem from "./BasketItem/index.jsx";
+
+const Basket = (props) => {
+
+    const { menu } = props
+    const { updateOrder } = props
+
+    const makeItem = () => {
+
+       let orderArray = [];
+       let itemPrice = 0;
+
+        for (let key in props.order) {
+            if (key !== 'foodPrice' && props.order[key] !== 0) {
```

Comment:
does key have to be called key here. Would it be more readable if we could use different names